src/main/kotlin/org/anefdev/mixbuddy/util/PlaylistParser.kt

```diff
@@ -0,0 +1,168 @@
+package org.anefdev.mixbuddy.util
+
+import org.anefdev.mixbuddy.model.MusicPlaylist
+import org.anefdev.mixbuddy.model.MusicTrack
+import org.springframework.stereotype.Component
+import se.michaelthelin.spotify.model_objects.miscellaneous.AudioAnalysis
+import se.michaelthelin.spotify.model_objects.specification.PlaylistSimplified
+import se.michaelthelin.spotify.model_objects.specification.PlaylistTrack
+import se.michaelthelin.spotify.model_objects.specification.Track
+import kotlin.math.abs
+
+@Component
```

Comment:
adding this class to spring context seems to be unnecessary as it only has util methods, its better to declare it as an `object` and call the methods directly